app.coffee

```diff
@@ -138,6 +138,15 @@ shutdownCleanly = (signal) ->
 			process.exit()
 		, 10000
 
+watchForEvent = (eventName)->
+	docUpdaterRedisClient.on eventName, (e)->
+		console.log "redis event: #{eventName} #{e}"
```

Comment:
This is clashing with the json output of the logger. What about using console.err to move the messages to stderr or better use logger.warn?

Reply:
Cheers. This is a bit of a temporary logging while we look into something, the console.log works well with our production logging environment.